Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,25 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def x11
+    xorg = Formula["linuxbrew/xorg/xorg"]
+    if xorg.installed?
+      xorg.recursive_dependencies.each do |d|
+        dep_formula = d.to_formula
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.lib/"pkgconfig"
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.share/"pkgconfig"
```

Comment:
At least for `core` formula, I prefer the tidiness of `depends_on :x11`, rather than depending on each individual subcomponent of X11 individually, and then is formula-compatible with Homebrew (not requiring Linuxbrew patches).